lookout/style/format/analyzer.py

```diff
@@ -233,15 +175,15 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_service: D
         _log.info("train %s %s %s %s", __version__, ptr.url, ptr.commit,
                   pformat(config, width=4096, compact=True))
         model = FormatModel().generate(cls, ptr)
-        config = cls._load_train_config(config)
+        config = cls._load_config(config)
         for language, files in files_by_language(files).items():
             try:
-                lang_config = config[language]
+                lang_config = config["train"][language]
             except KeyError:
                 _log.warning("language %s is not supported, skipped", language)
                 continue
-            _log.info("effective config for %s:\n%s", language,
-                      pformat(lang_config, width=120, compact=True))
+            _log.debug("effective train config for %s:\n%s", language,
```

Comment:
This *must* stay info. Please do not change unrelated code, otherwise I takes me more time to review and your PRS hang unmerged longer

Reply:
oops, it was unintentionally